.github/workflows/build-and-deploy-prod.yml

```diff
@@ -36,15 +37,17 @@ jobs:
         run: mvn package --settings .m2/maven-settings.xml --file pom.xml
       - name: Bygg Docker image
         run: |
-          docker build -t $IMAGE .
+          docker build -t ${IMAGE}:${TAG} -t ${IMAGE}:latest .
```

Comment:
Er vi helt sikre på at latest alltid er den versjonen vi er på jakt etter?

Reply:
For ende-til-ende så vil jeg anta det. Man kan jo ta en diskusjon på det. Her pusher vi begge taggene, så det er vel en mulighet for å bruke den versjonen man vil.